VideoLocker/src/test/java/org/edx/mobile/test/AuthenticationTests.java

```diff
@@ -102,6 +102,25 @@ public void testAuthenticate_notForExpiredAccessToken() throws Exception {
         assertEquals("401_not_caused_by_expired_token", response.request().header("Authorization"));
     }
 
+    @Test
+    public void testAuthenticate_withoutRefreshToken() throws Exception {
+        PrefManager pref = new PrefManager(context, PrefManager.Pref.LOGIN);
```

Comment:
Maybe we should have a list of classes that we should replace with injectable things?